### PR TITLE
5233: Changed last updated by to show the name of the user

### PR DESCRIPTION
**Pull request by sam-roberts-bjss**
Story: [AB#5322](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/5322)
Task: [AB#7052](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7052)

### CODE REVIEW

File: tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Orders GetAll 4826.feature [hunk 1]
```diff
@@ -4,42 +4,42 @@
...
+	Then a response with status code 403 is returned
 
 @4826
 Scenario: 5. Service Failure
-    Given the call to the database will fail
-    When a GET request is made for a list of orders with organisationId 4af62b99-638c-4247-875e-965239cd0c48
-    Then a response with status code 500 is returned
+	Given the call to the database will fail
+	When a GET request is made for a list of orders with organisationId 4af62b99-638c-4247-875e-965239cd0c48
+	Then a response with status code 500 is returned
```
Comment (RuthBerry96) [inline tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Orders GetAll 4826.feature:45]: I thought we discussed in code standards meetings or something like that to have a newline at the end of a feature file

File: tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderSteps.cs [hunk 2]
```diff
@@ -102,7 +104,10 @@ private sealed class OrdersTable
 
             public Guid LastUpdatedBy { get; set; }
 
+            public string LastUpdatedByName { get; set; }
+
             public DateTime LastUpdated { get; set; }
+
```
Comment (RuthBerry96) [inline tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderSteps.cs:110]: Newline not needed

File: tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Orders GetAll 4826.feature (hunk 1 above)
Comment (sam-roberts-bjss) [reply tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Orders GetAll 4826.feature:45]: 👍

File: tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderSteps.cs (hunk 2 above)
Comment (sam-roberts-bjss) [reply tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderSteps.cs:110]: 👍

**Pull request merged**